### PR TITLE
Show an error when sharing fails

### LINKED ISSUE #5504: Upload with no sharing rights
## Steps to reproduce
1. enable "exclude groups from sharing"
2. use such an user
3. upload a file -> see no meaningful error, but it just "hangs"

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #5504

### CODE REVIEW

File: src/components/MessagesList/MessagesGroup/Message/Message.vue [hunk 1]
```diff
@@ -544,6 +544,9 @@ export default {
 			if (this.sendingFailure === 'quota') {
 				return t('spreed', 'Not enough free space to upload file')
 			}
+			if (this.sendingFailure === 'failed-share') {
+				return t('spreed', 'You cannot share files')
```
Comment (PVince81) [inline src/components/MessagesList/MessagesGroup/Message/Message.vue:548]: why not "You are not allowed to share files" like below ? it's a bit clearer.

Comment (nickvergessen) [reply src/components/MessagesList/MessagesGroup/Message/Message.vue]: Done

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by tobiasKaminsky** [comment]
This is "only" about server side, or?
How are clients informed?

**Comment by nickvergessen** [comment]
well there is a 403 error and the web UI client just didn't handle it but swallowed all errors and thought it worked `¯\_(ツ)_/¯`

**Comment by tobiasKaminsky** [comment]
How do you handle those things? Shall I create an issue on Android and iOS then, or like files client one meta issue in this repo?

**Pull request merged**